src/_base/harness/scripts/mutagen.sh

```diff
@@ -33,7 +33,7 @@ install_mutagen()
         return 1
     fi
 
-    run curl -L -q -sS -f "$download_url" -o .my127ws/utilities/mutagen/mutagen.tar.gz
+    run curl --fail --silent --show-error --location --disable --output .my127ws/utilities/mutagen/mutagen.tar.gz "$download_url"
```

Comment:
I'm not sure -q was intentionally used to mean --disable, or mistaking for quiet?

Reply:
Mistaken for quiet 🤦 
I like the full flag naming instead of abbreviations 👍

Reply:
yes, it's generally recommended to use long options for scripting, and only use short options for personal usage